src/styles/blocks/card.scss

```
@@ -0,0 +1,129 @@
+.main {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $width;
+  border-radius: $border-radius;
+  border: $border;
+  font-family: Roboto, sans-serif;
+  transition: all 300ms;
```

It is better not to specify a transition for everything. Fix it in all code.

```suggestion
  transition: transform 300ms;
```